packages/modules/viewer-react/src/components/iModel/IModelLoader.tsx

```diff
@@ -62,7 +61,6 @@ const IModelLoader = React.memo(
     const { finalFrontstages, noConnectionRequired, customDefaultFrontstage } =
       useFrontstages(
         frontstages,
-        defaultUiConfig,
```

Comment:
do we have another way to hide the status bar? The CLI is currently using prop to hide it. The status bar provider just provides the items, but doesn't control visibility. Maybe we should just hide the bar when that provider isn't used?

Reply:
ah good catch, ill look into this

Reply:
the [StandardFrontstageProvider]() we use from core by default adds the navigation aid cube, the statusbar and the tool settings bar. you can optionally hide the navvigation aid and the status bar but not the tool settings. i can look into exposing an option to hide the tool settings bar in 3.3 but dont want to bump up the min core ver. I can re-add our own frontstage provider that forks what core has but adding that extra option. unofrutnately dont think there is an easy way to dynamically enable/disable these...
will continue investigating and update pr tomorrow

Reply:
There is a prop to hide the status bar  in the core front stage provider that I used in the cli pr. That part is fine. We just need a clean way to expose it if we remove this prop

Reply:
youre ok with not being able to hide the ToolSettingsBar at all?

Reply:
For now I think so, but we should add that option in the core frontstage

Reply:
added